code/modules/ruins/icemoonruin_code/hotsprings.dm

```diff
@@ -27,22 +27,7 @@ GLOBAL_LIST_EMPTY(cursed_minds)
 		return
 	GLOB.cursed_minds[L.mind] = TRUE
 	RegisterSignal(L.mind, COMSIG_PARENT_QDELETING, .proc/remove_from_cursed)
-	var/random_choice = pick("Mob", "Appearance")
-	switch(random_choice)
-		if("Mob")
-			L = wabbajack(L, "animal")
-		if("Appearance")
-			var/mob/living/carbon/human/H = wabbajack(L, "humanoid")
-			randomize_human(H)
-			var/list/all_species = list()
-			for(var/stype in subtypesof(/datum/species))
-				var/datum/species/S = stype
-				if(initial(S.changesource_flags) & RACE_SWAP)
-					all_species += stype
-			var/random_race = pick(all_species)
-			H.set_species(random_race)
-			H.dna.unique_enzymes = H.dna.generate_unique_enzymes()
-			L = H
+	L = wabbajack(L, "animal") // Appearance randomization removed so citadel players don't get randomized into some ungodly ugly creature and complain
```

Comment:
i feel like they'll get more pissed from being an animal but i guess whatever are you sure?

Reply:
not entirely but I feel like being a skub demon or whatever is probably less aggravating than getting turned into a citadel random appearance